docs/source/sysadmin_guide/pbft_updating_member_list.rst

```diff
@@ -0,0 +1,13 @@
+************************************
+PBFT Only: Updating PBFT Member List
+************************************
+
+If you are adding a new node to an existing PBFT network, you must update the
+on-chain setting ``sawtooth.consensus.pbft.members`` after the new node has
+been installed and configured. 
```

Comment:
Consider mentioning the setting takes effect after the containing block is published.

Reply:
Done.